### DISCUSSION TITLE
调用loadfile时的路径问题

### DISCUSSION THREAD

**Discussion by llxwd08** [starwing/lua-protobuf#178] [Q&A]
我的工程里 assert(pb.loadfile 'CProto.pb') 一直报错找不到CProto.pb文件，我是把CProto.pb和这句脚本的文件放在统一目录的。
如果换成assert(pb.load 'CProto.pb') 就没报错了，但按照范例里，加载.pb文件，应该是用pb.loadfile吧。
不太清楚这些load的路径应该怎么处理，毕竟在Unity中其他读取文件的代码都是封装后，使用AA异步读取了。

**Comment by starwing** [comment]
你有没有发现文档里都没有loadfile了……？

**Comment by llxwd08** [comment]
啊，我看pb.c在luaopen_pb里还有ENTRY(loadfile)啊，那这个loadfile还有什么用吗？实际上现在读取.pb文件也应该使用load了？

**Reply by starwing** [reply]
没写在文档里的意思是，如果不会用那还是别用了……

**Reply by starwing** [reply]
loadfile需要你自己很明白读文件到底是怎么用的，需要你知道当前的工作目录，而且手机端绝对是用不了的。所有这些解释起来太麻烦了，我就把接口从文档里删了。会用的可以继续用，不会用的自己想办法读了文件用pb.load
